src/services/expense.service.js

```diff
@@ -0,0 +1,62 @@
+const { Expense } = require('./../models/Expense.model');
+const { Op } = require('sequelize');
+const normalize = ({ amount, category, id, note, spentAt, title, userId }) => {
+  return {
+    amount,
+    category,
+    id,
+    note,
+    spentAt,
+    title,
+    userId,
+  };
+};
+
+const getAll = async ({ categories, from, to, ...query }) => {
+  const where = { ...query };
+
+  if (categories) {
+    where.category = categories;
```

Comment:
Ensure that `categories` is an array if you intend to filter by multiple categories. If it's a single category, this assignment will work, but for multiple categories, you might need to use `[Op.in]`.